provider/datagen/tests/data/postcard/fingerprints.csv

```diff
@@ -5649,4 +5649,4 @@ transliterator/rules@1, und-x-und-t-und-d0-test-m0-niels-s0-test, 1769B, 4540044
 transliterator/rules@1, und-x-und-t-und-d0-test-m0-rectesta-s0-test, 369B, 69c41d4b5c828833
 transliterator/rules@1, und-x-und-t-und-d0-test-m0-rectestr-s0-test, 237B, 3345ed066cbb729f
 transliterator/rules@1, und-x-und-t-und-latn-d0-ascii, 27083B, bb4fc0b86c032865
-units/info@1, und, 8946B, 3799c2f43c4acfe1
+units/info@1, und, 7910B, 297a0bc835ca3002
```

Comment:
@sffc and @robertbastian: 
Processing the base units reducing the data size from 8946B to 7910B ( saving 1036B).

Also, will reduce the processing time to extract the conversion rate too.